src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/AttachExceptionRecordToExistingCaseTest.kt

```diff
@@ -26,12 +29,43 @@ class AttachExceptionRecordToExistingCaseTest {
         RestAssured.requestSpecification = RequestSpecBuilder().setPort(applicationPort).setContentType(JSON).build()
     }
 
+
+    private val request = CallbackRequest.builder().eventId(CallbackTypes.ABOUT_TO_SUBMIT)
+
+    @Test
+    fun `should successfully callback with correct information`() {
```

Comment:
I thought we decided not to add more Kotlin code :)

Reply:
I'm just using the code I have already written for these previously